app/serializers/comment_serializer.rb

```diff
@@ -0,0 +1,11 @@
+class CommentSerializer <ActiveModel::Serializer
+  attributes :id, :author, :body, :created_at
+  
+  ANNONYMOUS = "annonymous"
+  
+def author
+  return ANNONYMOUS if object.author.blank?
+  
+  object.author
+end 
+end
```

Comment:
```suggestion
class CommentSerializer <ActiveModel::Serializer
  attributes :id, :author, :body, :created_at
  
  ANNONYMOUS = "annonymous"
  
  def author
    return ANNONYMOUS if object.author.blank?
  
    object.author
  end 
end

```

Please, fix syntax here